app/controllers/course_user_data_controller.rb

```diff
@@ -207,18 +227,18 @@ def add_users_breadcrumb
   end
 
   def cud_params
-    if @cud.administrator?
+    if @cud && @cud.administrator?
```

Comment:
Why are the @cud's being nil checked?  Will @cud ever not be defined in this controller?  Also, as a minor gripe, we should check for instructors before administrators, because the latter will require a user-load from the DB.  Why are we splitting the attributes if they are the exact same object anyway?